docs/_documentations/codewind-openapi-vscode.md

```diff
@@ -22,6 +22,21 @@ The Codewind OpenAPI Tools for VS Code provides commands that invoke the OpenAPI
 3. This extension pulls the [OpenAPI Generator CLI Docker Image](https://github.com/OpenAPITools/openapi-generator#16---docker) and runs the OpenAPI Generator in a Docker container. Install Docker if necessary.
 4. Install the Codewind OpenAPI Tools from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-openapi-tools) or by searching for `Codewind OpenAPI` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
 
+## Generating and building code in an existing Java Maven project
+1. Right-click your Codewind project and launch the **Generate Server from OpenAPI Definition** wizard.
+2. Select your OpenAPI document `.json` file.
+3. Select the `jaxrs-spec` server generator type.
+4. When you're prompted to select the output folder, keep the default selection. The project root, which is the parent of the source folder, is selected by default.
+5. Click **Select As Output Folder** to merge the generated `.pom` file with the existing `.pom` file. The merge attempts to include all necessary dependencies for the project to build properly.
```

Comment:
5. Click **Select As Output Folder**.  
Move the rest to after step 6.

Reply:
@keithchong It's better to include the results of an action as part of the step that includes the action. When you click **Select As Output Folder**, is that action what merges the `.pom` files? If so, it's probably best to leave these statements together.

Reply:
@sishida , that is the last step.  (Unlike in Eclipse, you can click on the Finish button of the wizard). Then the warning message appears right after.  If the user cancels the warning, then _no_ code generation and _no_ merging takes place.

Reply:
OK, I think I understand better what's going on. I will edit and add a new commit.